plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -338,7 +338,7 @@ protected CachingHiveMetastore(
         databaseCache = cacheFactory.buildCache(this::loadDatabase);
         tableNamesCache = cacheFactory.buildCache(this::loadAllTables);
         tablesWithParameterCache = cacheFactory.buildCache(this::loadTablesMatchingParameter);
-        tableStatisticsCache = statsCacheFactory.buildCache(this::loadTableColumnStatistics);
+        tableStatisticsCache = statsCacheFactory.buildCache(tableName -> new AtomicReference<>(loadTableColumnStatistics(tableName)));
```

Comment:
the loader defined here is never used (as the `tableStatisticsCache` is not used as a LoadingCache)

should't we use `buildBulkCache` here?

if not
- add a code comment why 
- use `tableName -> throw new Unsupported` as a loader here

Reply:
tableStatisticsCache is not bulk but maybe buildBulkCache is not named correctly. I will double-check that.
What we need here is a normal, not loading cache.